--- connections-41.0/src/rdp-connection.vala~	2021-09-17 12:16:22.000000000 +0000
+++ connections-41.0/src/rdp-connection.vala	2021-09-22 17:22:59.505140771 +0000
@@ -61,7 +61,7 @@
 
             display.rdp_connected.connect (() => { show (); });
             //display.rdp_needs_authentication.connect (on_rdp_auth_credential_cb);
-            display.rdp_auth_failure.connect (auth_failed);
+//            display.rdp_auth_failure.connect (auth_failed);
             //display.size_allocate.connect (scale);
 
             need_username = need_password = true;
